util/argo/normalizers/diff_normalizer.go

```diff
@@ -76,7 +76,13 @@ func NewIgnoreNormalizer(ignore []v1alpha1.ResourceIgnoreDifferences, overrides
 func (n *ignoreNormalizer) Normalize(un *unstructured.Unstructured) error {
 	matched := make([]normalizerPatch, 0)
 	for _, patch := range n.patches {
+		if patch.groupKind.Group == "" && patch.groupKind.Kind == "" {
```

Comment:
Kubernetes has an empty group, so we cannot assume that empty group value means any group. I suggest to support glob matching. This would also allow user to specify `*/*` as override key.

Example of using Glob library is here:
https://github.com/argoproj/argo-cd/blob/d222b935e6f5ec4d02eea2dc13870f7864945f3a/util/settings/filtered_resource.go#L24
https://github.com/argoproj/argo-cd/blob/d222b935e6f5ec4d02eea2dc13870f7864945f3a/util/settings/filtered_resource.go#L24